design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -223,6 +229,9 @@ type NegotiableQuote {
     status: NegotiableQuoteStatus!
 }
 
+type NegotiableQuoteShippingAddress implements NegotiableQuoteAddressInterface {
```

Comment:
Shipping method properties to be added as part of [PWA-2115](https://jira.corp.magento.com/browse/PWA-2115)